### PR TITLE
[AC-2444] feat: custom toggle button

### PR DESCRIPTION
**Pull request by bang9**
## Changes
- Updated path of files
- Updated `bot` and `chevron-down` icons
- Added `botStudioEditProps.styles.toggleButtonUrl`
- Added `toggle_button_url` to widget setting API and context

ticket: [AC-2444]

[AC-2444]: https://sendbird.atlassian.net/browse/AC-2444?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/widget/WidgetToggleButton.tsx [hunk 1]
```diff
@@ -78,21 +79,36 @@ const StyledArrowIcon = styled.span<{ isOpen: boolean }>`
...
 
+const Img = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  user-select: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -webkit-user-drag: none;
```
Comment (chohongm) [inline src/components/widget/WidgetToggleButton.tsx]: button 이라 cursor: pointer 를 추가하는게 좋지 않을까요?

**Review by chohongm — APPROVED** [review]
LGTM please check one comment and decide what to do and apply before merge if needed

File: src/components/widget/WidgetToggleButton.tsx (hunk 1 above)
Comment (bang9) [reply src/components/widget/WidgetToggleButton.tsx]: 요기는 버튼에 들어갈 이미지 입니닷

File: src/components/widget/WidgetToggleButton.tsx [hunk 2]
```diff
@@ -28,20 +28,42 @@ const StyledWidgetButtonWrapper = styled.button<{ accentColor: string }>`
...
     align-items: center;
 
     svg {
+      width: 32px;
+      height: 32px;
       path {
         fill: ${({ accentColor }) => getColorBasedOnSaturation(accentColor)};
       }
     }
+    img {
```
Comment (liamcho) [inline src/components/widget/WidgetToggleButton.tsx:48]: Can we split this into styled.img? for readability.

**Pull request merged**